File: app/src/main/java/com/ezequieljardim/recalibratedinclinometer/viewmodel/SensorViewModel.java
package com.ezequieljardim.recalibratedinclinometer.viewmodel;

import android.app.Application;
import android.arch.lifecycle.AndroidViewModel;

import com.ezequieljardim.recalibratedinclinometer.livedata.acceleration.AccelerationSensorLiveData;

public class SensorViewModel extends AndroidViewModel {
    private AccelerationSensorLiveData accelerationSensorLiveData;

    public SensorViewModel(Application application) {
        super(application);

        this.accelerationSensorLiveData = new AccelerationSensorLiveData(application);
    }

    public AccelerationSensorLiveData getAccelerationSensorLiveData() {
        return accelerationSensorLiveData;
    }
}

File: app/src/main/java/com/ezequieljardim/recalibratedinclinometer/viewmodel/SensorViewModel.kt
package com.ezequieljardim.recalibratedinclinometer.viewmodel

import android.app.Application
import androidx.lifecycle.AndroidViewModel
import com.ezequieljardim.recalibratedinclinometer.livedata.acceleration.AccelerationSensorLiveData

class SensorViewModel(application: Application?) : AndroidViewModel(application!!) {
    val accelerationSensorLiveData: AccelerationSensorLiveData = AccelerationSensorLiveData(application!!)

}